tests/console/switch_to_wicked.pm

```diff
@@ -0,0 +1,24 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Switch from NetworkManager to wicked.
+# Maintainer: Oleksandr Orlov <oorlov@suse.de>
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+
+sub run {
+    select_console 'root-console';
+    assert_script_run 'systemctl disable NetworkManager --now';
+    assert_script_run 'systemctl enable wicked --now';
```

Comment:
Let's add assertion that network is up and running, so that further modules can rely on that and do not perform additional checks. WDYT?

Reply:
I assumed, that if assert_script_run is not failed, then the command is executed properly. But, let's add additional check.

I've updated the Verification Run and it is passed.

Reply:
Yeah, the service managing network will be set, and I guess started, but it might fail to start, not sure if systemd will throw an error code in such case.